packages/paste-core/components/spinner/src/styled.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import {styled} from '@twilio-paste/styling-library';
+
+import {SvgKeyframes, CircleKeyframes} from './keyframes';
+
+type CircleStyleProps = Pick<React.SVGProps<SVGCircleElement>, 'strokeWidth' | 'fill' | 'strokeLinecap'> & {
+  transformOrigin: string;
+};
+
+const circleStyleProps: CircleStyleProps = {
+  fill: 'transparent',
+  strokeLinecap: 'round',
+  transformOrigin: '50% 50%',
+};
+
+export const StyledCircleTrack = styled.circle(() => {
+  return {
+    ...circleStyleProps,
+    stroke: 'currentColor',
+    opacity: 0.25,
+  };
+});
+
+export const AnimatedStyledCircle = styled.circle<{show: boolean}>(({show}) => {
+  return {
+    ...circleStyleProps,
+    animation: `1.4s linear infinite both ${CircleKeyframes}`,
+    strokeDasharray: '285',
+    stroke: 'currentColor',
+    '@media screen and (prefers-reduced-motion: reduce)': {
```

Comment:
I don't think we need this, since Paste's global styles does it for us now. You can test it by setting prefers-reduced-motion to on in your macbook accessibility settings.

Reply:
Ah ok cool! Wasn't sure about that. Thanks, will remove.